DigraphyApi/Repository/CollectionRepository.cs

```diff
@@ -0,0 +1,14 @@
+using DigraphyApi.Data;
+using DigraphyApi.Models;
+using DigraphyApi.Interfaces;
+using Microsoft.EntityFrameworkCore;
+
+namespace DigraphyApi.Repository;
+
+public class CollectionRepository(AppDbContext context) : ICollectionRepository
+{
+    public async Task<ICollection<Collection>> GetCollectionsAsync()
+    {
+        return await context.Collections.OrderBy(c => c.Id).ToListAsync();
```

Comment:
By default they're ordered by id I believe? Even if they're not I'm not sure we need to order them?